src/include/kernel/abi.h

```diff
@@ -29,7 +29,7 @@
 
 /** \brief SOF ABI version major, minor and patch numbers */
 #define SOF_ABI_MAJOR 3
-#define SOF_ABI_MINOR 14
+#define SOF_ABI_MINOR 15
```

Comment:
lets say 14, others pending on 14 too.

Reply:
OK, I'll remove ABI minor increase from next version. I'll still wait for new review round comments for kernel side to see if the changes were OK before I update this.

Reply:
Done